pkg/cvo/cvo.go

```diff
@@ -197,6 +204,20 @@ func (optr *Operator) InitializeFromPayload() error {
 
 	optr.releaseCreated = update.ImageRef.CreationTimestamp.Time
 	optr.releaseVersion = update.ImageRef.Name
+
+	// attempt to load a verifier as defined in the payload
+	verifier, err := verify.LoadFromPayload(update)
+	if err != nil {
+		return err
+	}
+	if verifier != nil {
+		glog.Infof("Verifying release authenticity: %v", verifier)
+	} else {
```

Comment:
would it be better if decided to use Reject based on error type vs nil verifier?

Reply:
I only did it for maximum paranoia (easier to review the code block here that guarantees that a verifier is set).